### ISSUE TITLE
"Special" characters not rendered properly

### ISSUE THREAD

**Issue by patrikalienus** [samhernandez/craft-shortcodes#10]
I'm using the following in one of my templates:

```twig
{{ entry.articleBody|shortcodes|raw }}
```

And it works fine mostly. What doesn't work for some reason, is the Swedish characters `Å`, `Ä`, `Ö`. Instead the "missing character" glyph is shown. The lowercase equivalents `å`, `ä`, `ö` work just fine. As soon as I remove the `|shortcodes` all characters are rendered properly again.

**Comment by samhernandez** [comment]
Thanks for the bug report. At first glance, it looks like a problem with the [thunderer/Shortcode](https://github.com/thunderer/Shortcode) package dependency. I'll take a look!

**Comment by engram-design** [comment]
I can't actually seem to reproduce this sorry! I'm using the following, just to test:

```twig
{{ 'The Swedish characters Å, Ä, Ö, The lowercase equivalents å, ä, ö work just fine.' | shortcodes }}

{# Also testing including raw, just in case #}
{{ 'The Swedish characters Å, Ä, Ö, The lowercase equivalents å, ä, ö work just fine.' | shortcodes | raw }}
```

Resulting in:
![image](https://user-images.githubusercontent.com/1221575/185823701-10bf1190-06e4-48dc-ac8d-32c2d1ae4703.png)

And just to make sure it's nothing to do with Redactor, I've also added that same string to Redactor and output it as you've shown.

I've also tested adding this text to the include for the shortcode (so `Testing [characters] content` - with the special characters in a `characters` shortcode include).
